tests/pytest/snapshot_test.py

```diff
@@ -0,0 +1,25 @@
+from test_dragonfly import DRAGONFLY_PATH, dfly_args, df_server, client, connection
```

Comment:
I imagine this is not the best way to structure our testing code. 
Should we use `conftest` for this? No need to do now, but please add a TODO that instructs on how to improve the code structure.

Reply:
Yep. I wasn't aware of `conftest`. I haven't used pytest much previously